projects/spectator/package.json

```diff
@@ -29,9 +29,9 @@
     "replace-in-file": "^4.1.3"
   },
   "peerDependencies": {
-    "@angular/common": "^9.0.1",
-    "@angular/core": "^9.0.1",
-    "@angular/router": "^9.0.1",
+    "@angular/common": ">= 8.0.0",
+    "@angular/core": ">= 8.0.0",
+    "@angular/router": ">= 8.0.0",
     "typescript": ">= 3.7.0"
```

Comment:
"typescript": ">= 2.8.0"

Reply:
let me check with this version of typescript,
I haven't checked it and not sure there are no new typescript features that are used in code which will not work with 2.8.0.

I will update you

Reply:
@NetanelBasal I would put "typescript": ">= 3.5.3" or smth like that b/c there are a bunch of error when using 2.8.0. Actually, 3.5.3 was used as dependency in the root package.json:
https://github.com/ngneat/spectator/commit/e13c9554778bdb179dfc7235aedb4b3b90302850#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L77

Here is a part of errors I'm getting with 2.8.0:
![image](https://user-images.githubusercontent.com/13370430/75884351-51f87c80-5e2d-11ea-8e9d-26114367672a.png)


Please confirm

Reply:
This was the lastest version before we release v5: https://github.com/ngneat/spectator/blob/5c98661cb61fe684f09b915b17629ca37749f867/projects/spectator/package.json#L35

Reply:
@NetanelBasal yes you're right, but what is goal of peerDependencies? Maybe I'm wrong but should not we specify versions which gonna work with other specified peerDependencies versions?
I assume that it was wrong with >= 2.8.0 but if you're ok I will get back to it.

Reply:
> Here is a part of errors I'm getting with 2.8.0:

What is the Angular version you're using?

Reply:
Let me check 2.8.0 with 8.0.0

Reply:
Doesn't work for me,
also 8.0.0 angular is using 3.4.2 version https://github.com/angular/angular/blob/38a7e2a775d2c3495a8574dde7d1a7106dd0ae04/package.json#L108

so I'm really unsure in 2.8.0 but pushed it. It doesn't break anything, just seems to be smth like disinformation.

Reply:
The TS version depends on your Angular version.

Reply:
Yes, that what I mean, and I'm not sure that TS 2.8.0 is compatible with Angular 8.0.0 :)